src/components/Botton.js

```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] As suggested by the previous reviewer kindly remove this disabling of the linter check. I removed it and it does not return any error - so it is redundant. Kindly remove it.

Reply:
- [ ] As mentioned by the last reviewer, Please remove this eslint disablement line. I removed it and it doesn't return any errors.